modules/cli/src/main/scala/scala/cli/commands/shared/ScalacOptions.scala

```diff
@@ -58,7 +58,13 @@ object ScalacOptions {
     * message instead.
     */
   val ScalacPrintOptions: Set[String] =
-    scalacOptionsPurePrefixes ++ Set("-help", "-Xshow-phases", "-Xplugin-list", "-Vphases")
+    scalacOptionsPurePrefixes ++ Set(
+      "-help",
+      "-Xshow-phases",
+      "-Xplugin-list",
+      "-Xlint:help",
+      "-Vphases"
```

Comment:
There are others, like `-opt:help`, `-Xsource:help`, `-Xmixin-force-forwarders:help`. Would it be possible to forward any `...:help`?

Or a more general change where the compiler is invoked in case there are no source files?

Somewhat related

```
➜ sandbox scala-cli compile --server=false -S 2 -Vprint-args a.txt A.scala
[error]  a.txt: unrecognized source type (expected .scala or .sc extension, or a directory).
If a.txt is meant to be treated as a script, add a shebang header in its top line.
  #!/usr/bin/env -S scala-cli shebang
When a shebang header is provided, the script can then be run with the 'shebang' sub-command, even if no file extension is present.
  scala-cli shebang a.txt
```